cmake-modules/gcc-arm-toolchain.cmake

```diff
@@ -1,3 +1,9 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
```

Comment:
Nice catch. Any other files in the repo that might be missing this license header?

Reply:
There is a cmake module in the form of a txt file inside eng/ folder.
This was created before using cmake modules. It can be updated later to be another cmake module.

Maybe yml files from eng/ should contain license as well